src/action.ts

```diff
@@ -111,22 +148,26 @@ async function linkAllFilesInMetafile(
 
 	if (decision === Decision.MATCH) {
 		return resultOf(linkExactTree(sourceRoot, fullLinkDir));
-	} else {
+	} else if (decision === Decision.MATCH_SIZE_ONLY) {
 		return resultOf(
 			fuzzyLinkOneFile(searchee, newMeta, fullLinkDir, sourceRoot)
 		);
+	} else {
+		return resultOf(
+			fuzzyLinkPartial(searchee, newMeta, fullLinkDir, sourceRoot)
+		);
 	}
 }
 
 export async function performAction(
 	newMeta: Metafile,
-	decision: Decision.MATCH | Decision.MATCH_SIZE_ONLY,
+	decision: Decision.MATCH | Decision.MATCH_SIZE_ONLY | Decision.MATCH_PARTIAL,
 	searchee: Searchee,
 	tracker: string
 ): Promise<ActionResult> {
 	const { action, linkDir } = getRuntimeConfig();
 
-	if (action === Action.SAVE) {
+	if (action === Action.SAVE || (decision === Decision.MATCH_PARTIAL && !linkDir)) {
```

Comment:
this should be a config schema validation, let's not let people get into this state in the first place.